bootstrap/roles/appliance-build.bootstrap/tasks/main.yml

```diff
@@ -37,10 +37,12 @@
     - qemu
     - zfsutils-linux
 
-- docker_image:
-    path: "{{ toplevel.stdout }}/docker"
-    name: appliance-build
-    force: true
+#
+# We can't use the docker_image module because it doesn't yet support passing
+# the 'network' parameter: https://github.com/ansible/ansible/pull/50313, which
+# we need to be able to fetch things from Artifactory.
+#
+- shell: docker build --network host --tag "appliance-build:latest" "{{ toplevel.stdout }}/docker"
```

Comment:
We need it so that we can reach other machines on the Delphix VPN, in particular, Artifactory. Do you feel that the comment that's there is incomplete?

Reply:
Oh, that's right.. the Dockerfile needs to reach out to Artifactory now, which is why we need the network parameter during the image build. Sorry; I just overlooked that detail, the comment is fine.